requirements.txt

```diff
@@ -44,11 +44,13 @@ boto3==1.26.60
 botocore==1.29.60
 clickhouse-connect==0.5.20
 db-dtypes==1.0.5
+google-api-core==2.11.0
 google-api-python-client==2.70.0
 google-cloud-bigquery~=3.0
 google-cloud-iam==2.9.0
 google-cloud-run==0.5.0
 google-cloud-storage==2.5.0
+google-cloud-pubsub==2.16.0
```

Comment:
We should add the new packages to this list: https://github.com/mage-ai/mage-ai/blob/master/setup.py#L99

Reply:
> We should add the new packages to this list: https://github.com/mage-ai/mage-ai/blob/master/setup.py#L99

Done.